### PR TITLE
OSDOCS-3262: Update Understanding the Compliance Operator with the latest information

### PR DESCRIPTION
**Pull request by kmccarron-rh**
https://issues.redhat.com/browse/OSDOCS-3262

Applies to 4.6+

https://deploy-preview-41597--osdocs.netlify.app/openshift-enterprise/latest/security/compliance_operator/compliance-operator-understanding.html

### CODE REVIEW

File: modules/compliance-profiles.adoc [hunk 1]
```diff
@@ -5,36 +5,56 @@
...
 
+* View the available profiles:
++
 [source,terminal]
 ----
 $ oc get -n <namespace> profiles.compliance
 ----
-
++
+This example displays the profiles in the default openshift-compliance namespace:
```
Comment (kmccarron-rh) [inline modules/compliance-profiles.adoc:17]: s/openshift-compliance/`openshift-compliance`/

File: modules/compliance-profiles.adoc [hunk 2]
```diff
@@ -45,113 +65,143 @@ description: |-
...
+* View the rules within a desired profile:
++
 [source,terminal]
 ----
-$ oc get -n <namespace> -oyaml rules.compliance <rule_name>
+$ oc get -n <namespace> -oyaml rules.compliance <rule_name> 
 ----
-
++
+This example displays the rhcos4-audit-rules-login-events rule in the rhcos4 profile:
```
Comment (kmccarron-rh) [inline modules/compliance-profiles.adoc:152]: Add tick marks around `rhcos4-audit-rules-login-events` and  `rhcos4`

File: modules/compliance-profiles.adoc [hunk 3]
```diff
@@ -5,36 +5,56 @@
...
-
+* View the details of a profile:
++
 [source,terminal]
 ----
 $ oc get -n <namespace> -oyaml profiles.compliance <profile name>
 ----
-
++
+This example displays the details of the rhcos4-e8 profile:
```
Comment (kmccarron-rh) [inline modules/compliance-profiles.adoc:51]: Ad tick marks around `rhcos4-e8`

File: modules/compliance-profiles.adoc [hunk 4]
```diff
@@ -45,113 +65,143 @@ description: |-
...
-    <li><code>audit_rules_login_events_faillock</code></li>
-    <li><code>audit_rules_login_events_lastlog</code></li></ul>
-    This rule checks for multiple syscalls related to login
-    events and was written with DISA STIG in mind.
-    Other policies should use separate rule for
-    each syscall that needs to be checked.
+  warning: Manual editing of these files may indicate nefarious activity, such as anattacker attempting to remove evidence of an intrusion.
 ----
 
-Each profile has the product name that it applies to added as a prefix to the profile's name. `ocp4-e8` applies the Essential 8 benchmark to the {product-title} product, while `rhcos4-e8` applies the Essential 8 benchmark to the {op-system-first} product.
```
Comment (kmccarron-rh) [inline modules/compliance-profiles.adoc:157]: This was moved to bottom of the first bullet.

**Review by kmccarron-rh — COMMENTED** [review]
Leaving comments for myself about adding tick marks.

**Review by jhrozek — APPROVED** [review]
Looks good from the technical perspective

File: modules/compliance-profiles.adoc [hunk 5]
```diff
@@ -45,113 +65,143 @@ description: |-
...
+      compliance.openshift.io/profile-bundle: rhcos4
+    name: rhcos4-e8
+    namespace: openshift-compliance
+    ownerReferences:
+    - apiVersion: compliance.openshift.io/v1alpha1
+      blockOwnerDeletion: true
+      controller: true
+      kind: ProfileBundle
+      name: rhcos4
+      uid: 6b6090b6-2dce-4f33-bbf9-e9ddd4c66f56
```
Comment (kmccarron-rh) [inline modules/compliance-profiles.adoc:87]: @pdhamdhe - Is the uid: line sensitive? Should it be removed from the example? Is there any other information that should be removed from the updated example output?

Comment (pdhamdhe) [reply modules/compliance-profiles.adoc:87]: @kmccarron-rh Hi, it's not sensitive but we can remove it to shorten the output along with that  remove `resourceVersion`, `creationTimestamp` & `generation` etc

**Comment by pdhamdhe** [comment]
LGTM

File: modules/compliance-profiles.adoc (hunk 5 above)
Comment (kmccarron-rh) [reply modules/compliance-profiles.adoc]: Done! Thanks for the info

**Comment by ahardin-rh** [comment]
Looks great!

**Pull request merged**